docs/kratos/mfa/15_totp.mdx

```diff
@@ -112,23 +122,33 @@ In this example, the user identifier (email - `alice@example.com`) is used. To l
    ory update identity-config <project-id> --file identity-config.yaml
    ```
 
+4. After TOTP has been enabled for the Ory Network project, each user is responsible for enabling TOTP in their own account by
+   pairing their account with a TOTP app.
+
+   :::note
+
+   In the Ory Account Experience, users are **not** automatically prompted to enable TOTP on their account. A user can enable TOTP
+   in their account by signing in and going to the **Authenticator App** section of the **Account Settings** page.
+
+   :::
+
 ```mdx-code-block
 </TabItem>
 </Tabs>
 ```
 
+## Custom identity schema
+
+By default, the identity schema is preconfigured to display the user's email address when requesting the TOTP short code. If you
+are working with a custom identity schema, however, you must ensure that the identity schema is configured to select an identity
+trait to display in the TOTP app.
```

Comment:
What do you mean by this? It's not clear to me. 
> identity schema is configured to select an identity trait to display in the TOTP app.
?

So basically exactly what trait is used to identify the user requesting for the code? That's described in the next paragraph `Distinguishing identities requesting TOTP`.

Reply:
Made this sentence more generic:
"If you are working with a custom identity schema, however, you must ensure that the identity schema is properly configured to work with TOTP."

With the details being left to the following paragraph.